src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -374,7 +374,7 @@ private static bool IsAbstractPropertyImplemented(INamedTypeSymbol classOrStruct
             ISymbol within)
         {
             var systemAttributeType = compilation.AttributeType();
-
+            var propertyNames = new HashSet<string>(StringComparer.Ordinal);
```

Comment:
Is there a standard way of passing this sort language specific information
into shared code? Or should it be adhoc?

On Mon, 8 Aug 2016 10:54 0xd4d, <notifications@github.com> wrote:

> In
> src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs
> <https://github.com/dotnet/roslyn/pull/12982#discussion_r73849625>:
>
> > @@ -374,7 +374,7 @@ private static bool IsAbstractPropertyImplemented(INamedTypeSymbol classOrStruct
> >              ISymbol within)
> >          {
> >              var systemAttributeType = compilation.AttributeType();
> > -
> > +            var propertyNames = new HashSet<string>(StringComparer.Ordinal);
>
> Visual Basic identifiers are case insensitive.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/roslyn/pull/12982/files/3479950d48893a913fcb4ba84575f06762e0e1ed#r73849625>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA822jhCw6Ncq-U9VS9juOzsrnWZdD-iks5qdvzZgaJpZM4JegCo>
> .
>